prow/cmd/deck/static/prow/prow.ts

```diff
@@ -670,7 +670,7 @@ function createRerunCell(modal: HTMLElement, rerunElement: HTMLElement, prowjob:
 
 function createViewJobCell(prowjob: string): HTMLTableDataCellElement {
     const c = document.createElement("td");
-    const i = icon.create("pageview", "Show job YAML");
+    const i = icon.create("pageview", "Show job YAML", "ga('send', 'event', 'view jow YAML', 'clicked');");
```

Comment:
Instead of pulling in and using the deprecated `ga`, instead [use `gtag`](https://developers.google.com/analytics/devguides/collection/gtagjs/events):

```ts
gtag('event', 'view_job_yaml', {event_category: 'engagement'})
```

However, that won't actually work reliably (and nor would the original) - because you're in the process of leaving the page, it is very possible that the page will be unloaded before the request can actually be sent. Fortunately, there is an API to deal with this ([`navigator.sendBeacon`](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/sendBeacon)), and we can instruct GA to use it by specifying `transport_type: "beacon"`:

```ts
gtag('event', 'view_job_yaml', {event_category: 'engagement', transport_type: 'beacon'})
```

This still doesn't work in older browsers, and there are solutions to that, but I don't think they're worth pursuing here.

Finally, having changed `create` to take a function instead of a string, you no longer need to include this in a string. The resulting code would look more like this:

```suggestion
    const i = icon.create("pageview", "Show job YAML", () => gtag("event", "view_job_yaml", {event_category: "engagement", transport_type: "beacon"}));
```